srt.go

```diff
@@ -202,6 +202,7 @@ func parseTextSrt(i string) (o Line) {
 				}
 
 				// Append item
+				s = unescapeWebVTT(s)
```

Comment:
Instead of adding this new line, could you unescape directly below like `Text: unescapeHTML(s),`?

Reply:
Done